core/services/chainlink/application.go

```diff
@@ -161,41 +161,27 @@ func (app *ChainlinkApplication) Start() error {
 		app.Exiter(0)
 	}()
 
-	ethEnabled := !app.Store.Config.EthereumDisabled()
-	if ethEnabled {
-		err := app.Store.EthClient.Dial(context.TODO())
-		if err != nil {
-			return err
-		}
-	}
-
 	// XXX: Change to exit on first encountered error.
 	return multierr.Combine(
+		app.Store.EthClient.Dial(context.TODO()),
```

Comment:
Interesting, should we have an application-wide context that gets "Done" on application close?

Reply:
Not sure, this was just moved around.